content/en/install.md

```diff
@@ -15,22 +15,11 @@ If you use `conda`, you can install it with:
 conda install numpy
 ```
 
-or with:
-
-```bash
-conda install -c conda-forge numpy
-```
-
 ## pip
 
 If you use `pip`, you can install it with:
 
 ```bash
 pip install numpy
 ```
-
-If your Python came with your operating system, we advise you either install a separate Python or use:
-
-```bash
-pip install numpy --user
-```
+If your Python is installed in a place where you don't have user-level write permissions (e.g. because it came with your operating system), we advise you either install a separate Python or use `pip install numpy --user`.
```

Comment:
As of pip 20.0 (released a couple of months ago), it will automatically detect when it needs to do a user installation based on permissions, so it shouldn't be necessary to mention this.

Of course, you may want to keep it for a while longer in case people still have older versions of pip, but as pip warns when it's outdated, hopefully most people will be reasonably up to date.

Reply:
Good point, I'll remove it. The case where `pip` can't be easily upgraded (Linux distros), numpy should anyway be installed with the distro package manager.